deploy/testnet/run-validator.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+# Runs a validator node:
+#
+# 1. Starts the archive node;
+# 2. Adds session keys;
+
+# expects files in the same directory:
+# - config.env
+# - chainspec.json
+
+# config.env should be created and uploaded by CD worker, it should contain:
+
+# BOOTNODE_ADDRESS=<node address in libp2p form>
+# PRIVATE_KEY=<key in hex>
+# DOCKER_IMAGE=<image name>
+
+source ./config.env
```

Comment:
We can use well-known bash trick to avoid having to call this script from specific _current working directory_. 
`"$(readlink -f "$0")"` expands into absolute path to this script file, `$(dirname "$(readlink -f "$0")")` into `testnet` absolute path for example.